robin/configuration.py

```diff
@@ -276,10 +272,10 @@ class Config:
         default=5, description="Number of candidates to generate for each query."
     )
     disease_name: str = Field(
-        default=DEFAULT_DISEASE_NAME, description="Name of the disease to focus on."
+        default="input_disease", description="Name of the disease to focus on."
     )
-    run_folder_name: str = Field(
-        default=DEFAULT_FOLDER_NAME,
+    run_folder_name: str | None = Field(
+        default=None,
```

Comment:
Can you briefly mention in the `description` what happens for the default of `None`, something like: `", default of None will autogenerate this from disease_name."`